src/multiroot.rs

```diff
@@ -0,0 +1,15 @@
+//
+// A rust binding for the GSL library
+//
+//! Multiroot test algorithms, See `rgsl::types::multiroot` for solvers.
```

Comment:
Please add an empty line before this one.

Reply:
added